cmd/datablue/handlers.go

```diff
@@ -451,6 +453,72 @@ func varsHandler(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprint(w, resp)
 }
 
+// apiHandler handles API requests which take the form:
+//
+//	/api/operation/property/value
```

Comment:
can we make some comment about what the path looks like after /value? ie chunk or even just "/.."